site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/service/SiteServiceImpl.java

```diff
@@ -92,7 +92,7 @@ private void addTypesToMap(final SiteConfiguration configuration) {
      * @return The id of the new site
      */
     @Override
-    public String save(final Site site) {
+    public String save(final uk.ac.ox.ndph.mts.site_service.model.Site site) {
```

Comment:
Could lose the package disambiguation if the other Site became SiteMessages?

Reply:
yep good call missed that IntelliJ had added the full package details to the Site model - Ill change